homeassistant/components/google_mail/strings.json

```
@@ -68,10 +68,10 @@
         },
         "restrict_domain": {
           "name": "Restrict to domain",
-          "description": "Restrict automatic reply to domain. This only affects GSuite accounts."
+          "description": "Restrict automatic reply to domain. This only affects Workspace accounts."
```

```suggestion
          "description": "Restrict automatic reply to domain. This only affects Google Workspace accounts."
```
Would this make more sense?

---

As a Google Workspace user and reseller myself we usually just call this "Workspace", but for more clarity – no problem. 😃